<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputSourceSelector</class>
 <widget class="QWidget" name="InputSourceSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>103</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QRadioButton" name="snesClassicRadioButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>SNES Classic</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">sourceRadioGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="snesClassicStuffRadioButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>SNES Classic (with mod)</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">sourceRadioGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="xinputRadioButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>PC (XInput)</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">sourceRadioGroup</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="infoLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>161</width>
     <height>47</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="arduinoRadioButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>SNES Arduino</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">sourceRadioGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="arduinoPortLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Port :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="arduinoComComboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="xinputComboBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>30</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="usb2snesRadioButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Usb2Snes</string>
   </property>
  </widget>
  <widget class="QComboBox" name="usb2snesComboBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>40</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="sourceRadioGroup"/>
 </buttongroups>
</ui>
